spec/bolt/module_installer/specs/git_spec_spec.rb

```diff
@@ -44,6 +44,15 @@
         /Invalid git source/
       )
     end
+
+    it 'errors if resolve is false and missing a name' do
```

Comment:
Since it's a separate check it'd also be good to verify that `resolve: <non-boolean>` errors for both forge and git modules.